CodeStyle/checkstyle.xml

```diff
@@ -45,17 +50,17 @@
         </module>
         <module name="RightCurly">
             <property name="option" value="same"/>
-            <property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY"/>
+            <property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, CLASS_DEF, METHOD_DEF, CTOR_DEF"/>
         </module>
         <module name="RightCurly">
             <property name="option" value="alone"/>
             <property name="tokens"
-                      value="CLASS_DEF, METHOD_DEF, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
+                      value="LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
         </module>
         <module name="OneStatementPerLine"/>
         <module name="EmptyLineSeparator">
             <property name="allowNoEmptyLineBetweenFields" value="true"/>
-            <property name="allowMultipleEmptyLines" value="false"/>
+            <property name="allowMultipleEmptyLines" value="true"/>
```

Comment:
What was the reason for allowing this? I note it is different from allowMultipleEmptyLinesInsideClassMembers.

Reply:
I changed it to true to align with the current state of the code. Looks like this rule has not been really enforced before upgrading to 8.29, so changing the value to `true` allows us to keep the code as it is (only imports are reorganized as I could not find proper setup for checkstyle.xml to keep imports as it is)